frame/nomination-pools/src/lib.rs

```diff
@@ -1288,6 +1288,7 @@ pub mod pallet {
 			pool_id: PoolId,
 			balance: BalanceOf<T>,
 			points: BalanceOf<T>,
+			era: EraIndex,
```

Comment:
Update the docs to say what is `era` please? is it the era in which we unbonded, or the one in which we will be unlocked?